aptos-move/framework/aptos-stdlib/sources/simple_map.move

```diff
@@ -105,34 +99,27 @@ module aptos_std::simple_map {
         (key, value)
     }
 
-    fun find<Key: store, Value: store>(
+    fun find_element<Key: store, Value: store>(
```

Comment:
why not just keep the name `find` and change implementation only? Otherwise you will break all the existing contracts on mainnet.

IMO just changing implementation is a natural choice because it's backwards compatible w/o any issues.